src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public abstract class Machine {
+    public abstract String goWork ();
+    public abstract String stopWork ();
```

Comment:
```suggestion
    public abstract void doWork()
    public abstract void stopWork ();
```

methods doWork() and stopWork() must be void